symengine/lib/symengine_wrapper.pyx

```diff
@@ -2977,11 +2979,16 @@ zoo = c2py(symengine.ComplexInf)
 nan = c2py(symengine.Nan)
 true = c2py(symengine.boolTrue)
 false = c2py(symengine.boolFalse)
+try:
+    import sympy
+    hbar = PyFunction(sympy.hbar, (), sympy.hbar.func, sympy_module)
```

Comment:
See my comment [here](https://github.com/symengine/symengine.py/pull/179#discussion_r130219504). Changing to `hbar` make no difference as well.